apps/explorer/src/pages/epochs/useEpochProgress.ts

```diff
@@ -2,18 +2,19 @@
 // SPDX-License-Identifier: Apache-2.0
 
 import { useTimeAgo } from '@mysten/core';
-import { useMemo } from 'react';
 
 export function useEpochProgress(
-    start?: number,
-    end?: number,
+    start: number = 0,
```

Comment:
Could this hook just fetch this data directly?

Reply:
yes - good call. i'll update